scripts/wikipedia_covid/timeseries/wiki_parser.py

```diff
@@ -0,0 +1,239 @@
+import datetime
```

Comment:
@edumorlom  can you modularize this and add tests? I'll review after!

Reply:
Done, thanks